setup.cfg

```diff
@@ -0,0 +1,10 @@
+[mypy]
```

Comment:
Is this section just to avoid merge conflicts with the mypy PR?

Reply:
mypy can read its config from either a .mypy.ini file, or from a [mypy] section of setup.cfg.

I'm deleting .mypy.ini file from the previous pr and copying that config here so everything is in one place.

Reply:
sgtm